lib/rubocop/cop/style/each_with_object.rb

```diff
@@ -33,7 +33,17 @@ def on_block(node)
           return unless first_argument_returned?(args, return_value)
           return if accumulator_param_assigned_to?(body, args)
 
-          add_offense(method, :selector, format(MSG, method_name))
+          add_offense(node, method.loc.selector, format(MSG, method_name))
```

Comment:
This was necessary so that the whole node is passed to the `autocorrect` method, giving us access to the block parameters (so that they can be reordered). The highlighted source is exactly the same as before.